.gitignore

```diff
@@ -117,3 +117,4 @@ dist
 *.whl
 .vscode
 docs/build
+pynumdiff/_version.py
```

Comment:
@Jacob-Stevens-Haas had a whole other `.gitignore` file for this, which I don't think is actually necessary.